spec/datadog/core/remote/component_spec.rb

```diff
@@ -258,8 +258,26 @@
 
     context 'with waiters' do
       it 'unblocks waiters' do
+        skip('Known flaky (assertion below sometimes fails with timeout)')
```

Comment:
Perhaps consider only skipping in CI, using `ENV['CI']`

Reply:
Up for debate, but in my view:

* If someone is running this locally, either they're trying to modify this class and they should be able to spot the skipped spec in the code with the huge comment or in the rspec output
* Or they don't care about this class and having this flaky spec impact them doesn't seem particularly useful either?

Reply:
(I'm going to mark this as resolved to be able to merge the PR, but I'm happy to discuss this and to open a follow-up PR if we want to add this change)